### PR TITLE
Update security scan docs

### PR DESCRIPTION
**Pull request by catherineluse**

### CODE REVIEW

File: content/rancher/v2.x/en/security/security-scan/_index.md [hunk 1]
```diff
@@ -31,36 +73,109 @@ Rancher can only run security scans on clusters that were created with RKE, whic
...
-1. From the cluster view in Rancher, click **Tools > CIS Scans.**
-1. Click the name of the report that has tests you want to skip.
-1. A **Skip** button is displayed next to each failed test. Click **Skip** for each test that should be skipped.
+Recurring scans can be scheduled to run on any RKE Kubernetes cluster.
 
-**Result:** The tests will be skipped on the next scan.
+To enable recurring scans, edit the advanced options in the cluster configuration during cluster creation or after the cluster has been created.
 
-To re-run the security scan, go to the top of the page and click **Run Scan.**
+To schedule scans for an existing cluster:
```
Comment (sowmyav27) [inline content/rancher/v2.x/en/security/security-scan/_index.md]: There is an "Add Schedule" option also in Global --> Tools --> CIS scans page which can be added here?

File: content/rancher/v2.x/en/security/security-scan/_index.md [hunk 2]
```diff
@@ -31,36 +73,109 @@ Rancher can only run security scans on clusters that were created with RKE, whic
...
+        "skip": { 
+            "rke-cis-1.4": [ "1.1.1", "1.2.2"] 
+        }
+    }
+}
+```
+
+**Result:** These tests will be skipped on subsequent scans that use the defined CIS Benchmark version.
+
+### Setting Alerts
```
Comment (sowmyav27) [inline content/rancher/v2.x/en/security/security-scan/_index.md]: There is an option similar to "Add Schedule" option in Global --> Tools --> CIS scans page which can be added here?

File: content/rancher/v2.x/en/security/security-scan/_index.md [hunk 3]
```diff
@@ -5,23 +5,65 @@ weight: 1
...
+
+The official Benchmark documents are available through the CIS website. The sign-up form to access the documents is [here.](https://learn.cisecurity.org/benchmarks)
 
 To check clusters for CIS Kubernetes Benchmark compliance, the security scan leverages [kube-bench,](https://github.com/aquasecurity/kube-bench) an open-source tool from Aqua Security.
 
 ### About the Generated Report
 
 Each scan generates a report can be viewed in the Rancher UI and can be downloaded in CSV format.
 
+As of Rancher v2.4, the CIS scan will use either the Benchmark v1.4 or v1.5, depending on the Kubernetes version.
```
[the remark this answers is not in the document]
Comment (catherineluse) [reply content/rancher/v2.x/en/security/security-scan/_index.md]: Updated

File: content/rancher/v2.x/en/security/security-scan/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/security/security-scan/_index.md]: Updated

File: content/rancher/v2.x/en/security/security-scan/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/security/security-scan/_index.md]: updated

**Pull request merged**